cpp/tests/utilities/column_utilities.cu

```diff
@@ -114,13 +114,16 @@ std::unique_ptr<column> generate_child_row_indices(lists_column_view const& c,
   //
   // result = [6, 1, 11, 1, 1]
   //
-  auto validity_iter = cudf::detail::make_counting_transform_iterator(
+  auto is_output_row = cudf::detail::make_counting_transform_iterator(
```

Comment:
What I was suggesting that this iterator and the one above (`row_size_iter`) are the same and should be combined.

Reply:
reused `row_size_iter` and use a stencil predicate in `scatter_if`.